mindsdb/interfaces/storage/db.py

```diff
@@ -115,6 +115,8 @@ class Predictor(Base):
     training_data_rows_count = Column(Integer)
     training_start_at = Column(DateTime)
     training_stop_at = Column(DateTime)
+    label = Column(String, nullable=True)
+    version = Column(Integer, default=1)
```

Comment:
I think that we shouldn't store a model version just as `int`. More proffessional approach is to store it as string in sematic versioning format https://semver.org/

Reply:
it our case version is autoincremented integer